suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -80,6 +81,9 @@ export const XpubScanScreen = ({
     };
 
     const handleRequestCamera = () => {
+        reset({
```

Comment:
Why is this better than setValue?

Reply:
I follow this statement in the react-hook-form Q&A (https://github.com/react-hook-form/react-hook-form/discussions/7671)

- also `reset` will not only reset your form values, but it'll wipe the form's states like _isDirty, isValid_ etc.